banzai_nres/tests/test_qc_wavelength.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import mock
+import pytest
+
+from banzai_nres.qc.qc_wavelength import AssessWavelengthSolution
+from banzai_nres.tests.test_wavelength import TestWavelengthCalibrate
+from banzai import context
+
+
+class TestAssessWavelengthSolution:
+    test_image = TestWavelengthCalibrate().generate_image()
+    input_context = context.Context({})
+    test_qc_results = {}
+
+    def test_do_stage_does_not_crash(self):
+        image = AssessWavelengthSolution(self.input_context).do_stage(self.test_image)
+        assert image is not None
+
+    @pytest.mark.integration
```

Comment:
At some level this is an integration test, but we haven't really marked other integration tests so adding more labels may or may not be helpful here.